pkg/oci/containerd.go

```diff
@@ -567,6 +566,35 @@ func AddMirrorConfiguration(ctx context.Context, fs afero.Fs, configPath string,
 	return nil
 }
 
+func getHostFile(fs afero.Fs, appendToBackup bool, registryURL url.URL, configPath string) (hostFile, error) {
+	if appendToBackup {
+		fp := path.Join(configPath, backupDir, registryURL.Host, "hosts.toml")
+		b, err := afero.ReadFile(fs, fp)
+		if err != nil && !errors.Is(err, afero.ErrFileNotFound) {
+			return hostFile{}, err
+		}
+		if err == nil {
+			hf := hostFile{}
+			err := toml.Unmarshal(b, &hf)
+			if err != nil {
+				return hostFile{}, err
```

Comment:
Log when files are appended to instead of created to make it clear to the end user.